[Bug]:Break after version change

## Bug Report

## Current behavior

Break on npm install after version change

## Input Code

<!-- REPL or Repo link if applicable: -->

```ts
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: nest-starter@0.0.1
npm ERR! Found: @nestjs/platform-socket.io@8.0.6
npm ERR! node_modules/@nestjs/platform-socket.io
npm ERR!   @nestjs/platform-socket.io@"^8.0.6" from the root project
npm ERR!   peerOptional @nestjs/platform-socket.io@"^8.0.0" from @nestjs/websockets@8.0.6
npm ERR!   node_modules/@nestjs/websockets
npm ERR!     @nestjs/websockets@"^8.0.6" from the root project
npm ERR!     peer @nestjs/websockets@"^8.0.0" from @nestjs/platform-socket.io@8.0.6
npm ERR!     2 more (@nestjs/core, @nestjs/microservices)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @nestjs/platform-socket.io@"^7.0.0" from @ogma/platform-socket.io@3.0.1
npm ERR! node_modules/@ogma/platform-socket.io
npm ERR!   @ogma/platform-socket.io@"^3.0.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/zieglar/.cache/node-v14.17.5/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/zieglar/.cache/node-v14.17.5/_logs/2021-09-13T04_19_14_358Z-debug.log
```

## Expected behavior

<!-- A clear and concise description of what you expected to happen (or code). -->

## Possible Solution

<!--- Only if you have suggestions on a fix for the bug -->

## Environment

<pre><code>
@ogma/nestjs-module version: 3.1.0

For Tooling issues:
- Node version: 14.17.5
- Platform:  Mac

Others:
<!-- Anything else relevant?  Operating system version, IDE, package manager, ... -->
</code></pre>

--- a/.changeset/eight-dolls-smash.md
+++ b/.changeset/eight-dolls-smash.md
@@ -1,0 +1,5 @@
+---
+'@ogma/platform-socket.io': patch
+---
+
+Fix the peer dependencies for socket.io and @nestjs/plaltform-socket.io

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -38,23 +38,23 @@ jobs:
         uses: changesets/action@master
         with:
           # This expects you to have a script called release which does a build for your packages and calls changeset publish
-          publish: pnpm release
+          publish: sed -i -e "s/'packages/'dist/" pnpm-workspace.yaml && pnpm release
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-  agent_1:
-    runs-on: ubuntu-latest
-    name: Agent 1
-    timeout-minutes: 60
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 14.x
-      - name: Install pnpm
-        run: npm i -g pnpm
-
-      - name: Install Dependencies
-        run: pnpm i --frozen-lockfile=false
-      - run: npx nx-cloud start-agent
+  #  agent_1:
+  #    runs-on: ubuntu-latest
+  #    name: Agent 1
+  #    timeout-minutes: 60
+  #    steps:
+  #      - uses: actions/checkout@v2
+  #      - uses: actions/setup-node@v1
+  #        with:
+  #          node-version: 14.x
+  #      - name: Install pnpm
+  #        run: npm i -g pnpm
+          #
+  #      - name: Install Dependencies
+  #        run: pnpm i --frozen-lockfile=false
+  #      - run: npx nx-cloud start-agent

--- a/package.json
+++ b/package.json
@@ -42,7 +42,7 @@
     "test:int": "nx run-many --target=e2e --all",
     "sendCoverage": "codeclimate-test-reporter < coverage/lcov.info",
     "prepare": "husky install",
-    "release": "nx affected --target=publish",
+    "release": "pnpm changeset",
     "nx": "nx",
     "deploy": "nx deploy docs"
   },

--- a/packages/platform-socket.io/package.json
+++ b/packages/platform-socket.io/package.json
@@ -24,10 +24,10 @@
     "url": "https://github.com/jmcdo29/ogma/issues"
   },
   "peerDependencies": {
-    "@nestjs/platform-socket.io": "^7.0.0",
-    "@nestjs/websockets": "^7.0.0",
+    "@nestjs/platform-socket.io": "^8.0.0",
+    "@nestjs/websockets": "^8.0.0",
     "@ogma/nestjs-module": "^3.0.0",
-    "socket.io": "^2.0.0"
+    "socket.io": "^4.0.0"
   },
   "peerDependenciesMeta": {
     "@nestjs/websockets": {

